### PR TITLE
Adding design inspiration resources

### PR DESCRIPTION
**Pull request by kvenn**
Mobile Patterns, UI Garage, Mobbin, scrnshts, and dribbble

There weren't many resources in the design inspiration category so I added a few. All of which are free and somewhat different (and with different catalogs of content).

If you'd like me to remove any, I'd be happy to do so. I took the description from their pages directly.

### CODE REVIEW

**Comment by ripienaar** [comment]
Seems fine, you could also add a category for them since we now have quite a few?

Let me know if you rather not and I'll merge, thanks

**Comment by kvenn** [comment]
Just created a new section and added a few more good, free resources. Let me know if this is what you had in mind!

File: README.md [hunk 1]
```diff
@@ -1219,6 +1221,18 @@ Table of Contents
 
 **[⬆ back to top](#table-of-contents)**
 
+## Design Inspiration
```
Comment (ripienaar) [inline README.md:1224]: Very good, please add a blank line below the heading here and we're good to go! thanks for the great improvement and new entries

**Comment by ripienaar** [comment]
thank you!

**Pull request merged**